README.md

```diff
@@ -61,17 +61,26 @@ receive reports from the car and send spoofed commands.
 
 ## Pre-requisites
 
-You must have Arduino Core and CMake (version 2.8 or greater) installed on
+You must have Arduino Core (version 1.8.5 or greater) and CMake (version 2.8 or greater) installed on
 your machine.
 
 ```
-sudo apt install arduino-core build-essential cmake
+sudo apt install build-essential cmake
+```
+
+A manual Arduino install is required since the debian package is older than 1.8.5
+```
+wget http://arduino.cc/download.php?f=/arduino-1.8.5-linux64.tar.xz -O arduino-1.8.5.tar.xz
+tar -xf arduino-1.8.5-linux64.tar.xz
+cd arduino-1.8.5
+sudo mkdir /usr/share/arduino
```

Comment:
this should be `sudo mkdir -p`. When I ran it, the directory still existed so it didn't need to be made again.